src/chains-config/cache/lruCache.ts

```diff
@@ -0,0 +1,11 @@
+import LRU from 'lru-cache';
+
+import { IBlock } from '../../types/responses';
+
+export const initLRUCache = (): LRU<string, IBlock> => {
+	const config = { max: 2 };
```

Comment:
what does `max == 2` here? Max two blocks can be cached in memory?

Reply:
Yup, exactly. The max length of the cache wont exceed 2. This is actually kinda up in the air, but from a brief discussion with Zeke we figured 1-2 would be more sufficient for the needs of polling `blocks/head`.